operator/config/crd/bases/forklift.konveyor.io_plans.yaml

```diff
@@ -730,6 +733,9 @@ spec:
                           description: 'The object ID. vsphere: The managed object
                             ID.'
                           type: string
+                        multiBootDiskNumber:
```

Comment:
do we need these?

I though we only need the one in the `VM` struct

Reply:
@bennyz probably right, it was automatically generated.